src/styles/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: $card-width;
+  border: $card-border;
+  border-radius: 5px;
+  box-sizing: border-box;
+  cursor: pointer;
+  transition-duration: $transition-duration;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    height: 134px;
+    width: 160px;
+    background-size: 160px 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    color: $color-main;
```

Comment:
Ensure that the variable `$color-main` is defined in your SCSS files. If it's not defined, this will cause a compilation error.